src/Workspaces/Core/Portable/Workspace/Solution/TextDocumentState.cs

```diff
@@ -369,15 +369,15 @@ public TextDocumentState UpdateText(TextLoader loader, PreservationMode mode)
                 lazyChecksums: null);
         }
 
-        private async Task<TextAndVersion> GetTextAndVersionAsync(CancellationToken cancellationToken)
+        private Task<TextAndVersion> GetTextAndVersionAsync(CancellationToken cancellationToken)
```

Comment:
💭 Not sure about this one. Maybe it's easy to understand in call stacks but maybe not.